### DISCUSSION TITLE
New Mini Dexed tutorial, with Hapax and Midi Fighter

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#719] [Show and tell]
If you understand french, i've made a video to explain my Mini Dexed setup.

https://youtu.be/OOt0OFQybvs?si=Fgj09kUzh8paxJOR

**Comment by Banana71** [comment]
Unfortunately I didn't understand anything but I knew what it was about :-)

Thanks for the great video.

Peter

**Reply by probonopd** [reply]
Reminds me of [this](http://infolab.stanford.edu/pub/voy/museum/pictures/display/0-4-Google.htm), in a good way. 👍 
Thanks for sharing.
